doc/async_worker.md

```diff
@@ -194,16 +192,15 @@ operations ends. The given function is called from the main event loop thread.
 identifier for the kind of resource that is being provided for diagnostic
 information exposed by the async_hooks API.
 
-Returns an AsyncWork instance which can later be queued for execution by calling
+Returns a `Napi::AsyncWork` instance which can later be queued for execution by calling
 `Queue`.
 
-
 ### Constructor
 
-Creates a new `AsyncWorker`.
+Creates a new `Napi::AsyncWorker`.
 
 ```cpp
-explicit AsyncWorker(const Object& receiver, const Function& callback, const char* resource_name, const Object& resource);
+explicit Napi::AsyncWorker(const Object& receiver, const Function& callback, const char* resource_name, const Object& resource);
```

Comment:
nit: The types of the formal arguments should probably be namespaced as well.

Reply:
Done